totalRP3/core/impl/utils.lua

```diff
@@ -441,10 +448,15 @@ local escapes = {
 }
 function Utils.str.sanitize(text)
 	if not text then return end
+	local originalText = text;
 	for k, v in pairs(escapes) do
 		text = text:gsub(k, v);
 	end
-	return text;
+	-- Looping for nested tags
+	if originalText ~= text then
+		text = Utils.str.sanitize(text)
+	end
+	return text
```

Comment:
I'd feel better if this were a loop rather than recursion due to stack overflow potential, even if exceptionally unlikely due to string length limitations.

```suggestion
function Utils.str.sanitize(text)
	if not text then return end
	-- Repeat until nested tags are eliminated.
	repeat
		local originalText = text;
		for k, v in pairs(escapes) do
			text = text:gsub(k, v);
		end
	until originalText == text;
	return text
```